cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/TokenPartitioner.java

```diff
@@ -138,15 +143,15 @@ private void validate()
     private void validateRangesDoNotOverlap()
     {
         List<Range<BigInteger>> sortedRanges = partitionMap.asMapOfRanges().keySet().stream()
-                .sorted(Comparator.comparing(Range::lowerEndpoint))
-                .collect(Collectors.toList());
+                                                           .sorted(Comparator.comparing(Range::lowerEndpoint))
+                                                           .collect(Collectors.toList());
         Range<BigInteger> previous = null;
         for (Range<BigInteger> current : sortedRanges)
         {
             if (previous != null)
             {
                 Preconditions.checkState(!current.isConnected(previous) || current.intersection(previous).isEmpty(),
-                        String.format("Two ranges in partition map are overlapping %s %s", previous, current));
+                                         String.format("Two ranges in partition map are overlapping %s %s", previous, current));
```

Comment:
since this line is changed.. You can use the other checkState api to lazily evaluate the error message only when the check fails. 

```suggestion
                                         "Two ranges in partition map are overlapping %s %s", previous, current);
```